### PR TITLE
Add CLI option to allow external entities

### PR DESCRIPTION
**Pull request by doshmajhan**
This is a fix for the recent change that got merged in that was failing the tests. The main reason the tests were failing is a few of the tests were trying to convert all the SVGs in the directory, and the new SVG I had added with the external resource was getting lumped in there causing it to fail when trying to convert it. So I've separated the two types of sample SVGs into their own directories and the tests passed again. 

I also realized it may be valid for a user to want to convert an SVG with an external resource defined in it, so I've added a CLI argument to allow this to happen when passed. The default still being to disallow external resources.

### CODE REVIEW

File: src/main/java/org/sterl/svg2png/config/OutputConfig.java [hunk 1]
```diff
@@ -37,7 +38,7 @@ public void addOutput() {
         files.add(new FileOutput());
     }
     
-    public boolean hasDirctoryOrFile() {
+    public boolean hasDirectoryOrFile() {
```
Comment (doshmajhan) [inline src/main/java/org/sterl/svg2png/config/OutputConfig.java:41]: Fixing a small typo here.

**Comment by doshmajhan** [comment]
Never mind I see you already did this, beat me to it!

File: src/main/java/org/sterl/svg2png/config/OutputConfig.java (hunk 1 above)
Comment (sterlp) [reply src/main/java/org/sterl/svg2png/config/OutputConfig.java:41]: I will merge this

**Comment by sterlp** [comment]
I like your naming better than mine, I think I will manually overtake some of your changes if this is fine with you. Not sure how I can credit you for your changes // basically what was part of the PR. I added your name to the release. Hints welcome :-)

**Comment by doshmajhan** [comment]
> I like your naming better than mine, I think I will manually overtake some of your changes if this is fine with you. Not sure how I can credit you for your changes // basically what was part of the PR. I added your name to the release. Hints welcome :-)

Ah, don't worry about credit at all, having my name in the release works for me. Go ahead and make the change!

**Comment by sterlp** [comment]
done: https://github.com/sterlp/svg2png/commit/4b1cebf311d43bb60fee97cf9a669ec1e795476f

**Pull request closed without merging**